integration_test/testutil_test.go

```diff
@@ -205,48 +205,46 @@ func (it *ITest) WithStdin(r io.Reader) *ITest {
 	return it
 }
 
-// RunOrFail runs the krew command and fails the test if the command returns an error.
-func (it *ITest) RunOrFail() {
-	it.t.Helper()
-	if err := it.Run(); err != nil {
-		it.t.Fatal(err)
-	}
-}
-
-// Run runs the krew command.
-func (it *ITest) Run() error {
+// Run runs the krew command, and returns its combined output even when
+// it fails.
+func (it *ITest) Run() ([]byte, error) {
```

Comment:
```suggestion
func (it *ITest) Run() (string, error) {
```
I could be mistaken, but I think we literally _everywhere_ convert the returned byte array to a string. So we could just move the conversion into this helper.

Reply:
`RunOrFailOutput` is `[]byte` so I kept it consistent w/ that.
We can convert both to `string` in another PR, wanted to keep this patch short.